### PR TITLE
Changes to fetch accounts from Plaid API #50

### PR DESCRIPTION
**Pull request by azc242**
Related to #50. The changes allow our frontend to fetch accounts from Plaid.

Some changes to visibly see the bank accounts fetched from Plaid API:
image
(Yes, it looks kind of empty. I only have 1 bank account 😢 )

Note that this PR does not cover the account details page. That will be a separate PR.

### CODE REVIEW

File: front-end/src/components/Accounts/AccountsPage.js [hunk 1]
```diff
@@ -56,37 +56,47 @@ const PlaidLink = (props) => {
     3. setBankDetailName (function to pass as prop to AccountPanel)
  */
 function AccountsPage(props) {
-  const data = props.banks;
   const [token, setToken] = useState(null);
+  const [data, setData] = useState([])
```
Comment (Jennifercheukyin) [inline front-end/src/components/Accounts/AccountsPage.js:60]: naming: bankData or something with bank prefix

File: back-end/app.js [hunk 2]
```diff
@@ -135,5 +156,28 @@ app.post('/api/set_access_token', async (request, response, next) => {
   }
 })
 
+app.post('/api/get_bank_accounts', async (request, response, next) => {
```
Comment (Jennifercheukyin) [inline back-end/app.js:159]: just think of this, could you add the web link of the api you are using? will be helpful in reviewing pr to see the request and response type.

File: front-end/src/components/Accounts/AccountsPage.js (hunk 1 above)
Comment (azc242) [reply front-end/src/components/Accounts/AccountsPage.js]: fixed! :)

File: back-end/app.js (hunk 2 above)
Comment (azc242) [reply back-end/app.js:161]: good idea, added into lastest commit

**Review by Jennifercheukyin — APPROVED** [review]
LGTM, thanks for the update! gooood job

File: back-end/app.js [hunk 3]
```diff
@@ -134,6 +155,31 @@ app.post("/api/set_access_token", async (request, response, next) => {
...
+
+    ACCESS_TOKEN = obj.access_token
+    const res = await plaidClient.accountsGet({
+      access_token: ACCESS_TOKEN,
+    });
+    const accounts = res.accounts;
+    return response.json(constructAccountsArr(res.data.accounts));
+  } catch (error) {
+    console.log("ERROR:");
+    prettyPrintResponse(error.response)
```
Comment (tangym27) [inline back-end/app.js:176]:
```suggestion
    prettyPrintResponse(error)
```

**Review by tangym27 — APPROVED** [review]
looks great – just make to make that change so that things don't crash when users initially dont have a bank account

**Pull request merged**